--- library/kmid.kcfg.new	2010-03-15 13:41:55.000000000 +0100
+++ library/kmid.kcfg	2010-03-15 13:42:23.000000000 +0100
@@ -100,7 +100,7 @@
 
     <entry name="exec_timidity" type="Bool">
       <label>Run TiMidity++ at startup</label>
-      <default>false</default>
+      <default>true</default>
     </entry>
     <entry name="cmd_timidity" type="Url">
       <label>Command</label>
@@ -114,7 +114,7 @@
         <choice name="oss"/>
         <choice name="jack"/>
       </choices>
-      <default>alsa</default>
+      <default>pulseaudio</default>
     </entry>
     <entry name="audiodev_timidity" type="String">
       <label>Audio Device</label>
